docs/docs/api/LayoutAnimations/SharedElementTransition.md

```diff
@@ -0,0 +1,113 @@
+---
+id: sharedElementTransition
+title: Shared Element Transition
+sidebar_label: Shared Element Transition
+---
+
+:::caution
+
+The Shared Element Transition is an experimental feature. No recommended for production usage. We are waiting for your feedback to improve implementation.
+
+:::
+
+A Shared Element Transition allows you to smoothly transform a component from one screen into a component on another screen. When Reanimated detects that a component with a `sharedTransitionTag` is being mounted or unmounted, it tries to find the last registered view with the same `sharedTransitionTag`. If it finds two matching components, it takes a snapshot of the styles for both components, and both shared views are detached from their parent and attached to a temporary transition container for the duration of the animation. After the animation is complete, they are attached back to their original parent. If you do not create a custom animation, all snapshot properties, including `width`, `height`, `originX`, `originY`, and `transformMatrix`, are animated by default with a duration of 500ms using the `withTiming` animation.
+
+## How to use it?
+To create a shared transition animation between two components on different screens, simply assign the same `sharedTransitionTag` to both components. When you navigate between screens, the shared transition animation will automatically play. The shared transition feature works by searching for two components that have been registered with the same `sharedTransitionTag`. If you want to use more than one shared view on the same screen, be sure to assign a unique shared tag to each component.
+
+**Screen A**
+```tsx {2}
+<View 
+  sharedTransitionTag="reanimatedTransition" // <-- shared tag
```

Comment:
1. Let's use `sharedTag` here for the purpose of consistency.
2. This line is already highlighted so we don't need the arrow in the comment.
```suggestion
  sharedTransitionTag="sharedTag"